solutions/aspnet/Makefile

```diff
@@ -4,11 +4,11 @@ include $(TOP)/defs.mak
 APPBUILDER=$(TOP)/scripts/appbuilder
 APPNAME=webapp
 
-#ifdef STRACE
-#OPTS += --strace
-#endif
+ifdef STRACE
+OPTS += --strace
+endif
 
-OPTS = --memory-size=256m
+OPTS += --memory-size=2048m
```

Comment:
This setting should has no effect as the same setting in config.json takes precedence.